combat-trainer.lic

```diff
@@ -2753,7 +2763,7 @@ class AttackProcess
       end
       game_state.loaded = false
       waitrt?
-    elsif game_state.loaded && !game_state.aiming_trainables.empty?
+    elsif game_state.loaded && !game_state.aiming_trainables.empty? #&& (Time.now - @firing_timer).to_i < @firing_delay
```

Comment:
❓ Please either uncomment this or remove the comment

Reply:
I was testing if it was needed. I do not believe it is and can remove.